shunting-yard-replicator/pom.xml

```diff
@@ -96,6 +97,16 @@
       <groupId>org.hibernate</groupId>
       <artifactId>hibernate-validator</artifactId>
     </dependency>
+    
+   <dependency>
```

Comment:
why are these need?

Reply:
Javax ExpressionFactory games.

You get `Caused by: java.lang.NoSuchMethodError: javax.el.ExpressionFactory.newInstance()` error while running test otherwise.